### PR TITLE
New package: prometheus-nginx-exporter

### PR DESCRIPTION
**Pull request by benley**
Adding to the growing collection of prometheus tools...

### CODE REVIEW

File: pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ lib, goPackages, fetchFromGitHub }:
+
+goPackages.buildGoPackage rec {
+  name = "prometheus-nginx-exporter-${builtins.substring 0 7 rev}";
```
Comment (jagajaga) [inline pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix]: https://github.com/NixOS/nixpkgs/blob/master/doc/coding-conventions.xml#L259
Read this please.

File: pkgs/top-level/go-packages.nix [hunk 2]
```diff
@@ -1991,6 +1991,19 @@ let
     buildInputs = [ protobuf ];
   };
 
+  prometheus.log = buildGoPackage rec {
+    name = "prometheus-log-${stdenv.lib.strings.substring 0 7 rev}";
```
Comment (jagajaga) [inline pkgs/top-level/go-packages.nix]: Same here.

File: pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix (hunk 1 above)
Comment (nckx) [inline pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix]: Are you aware of [this PR](https://github.com/NixOS/nixpkgs/pull/6539)?

(If so, fine. I'm not the version police.)

(Actually that sounds cool and I am now the version police. :cop: )

Comment (jagajaga) [inline pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix]: Surely I know this PR, I've made it.

Comment (nckx) [inline pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix]: My message was addressed to the author. There was no-one else.

I'm so fed up with GitHub.

Comment (jagajaga) [inline pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix]: @nckx Sorry.

File: pkgs/top-level/go-packages.nix (hunk 2 above)
Comment (benley) [inline pkgs/top-level/go-packages.nix]: done and done.

**Pull request merged**